qa/tasks/cephfs/test_volumes.py

```diff
@@ -2530,3 +2530,212 @@ def test_subvolume_snapshot_clone_cancel_pending(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+    def test_subvolume_ops_on_nonexistent_vol(self):
+        # tests the fs subvolume operations on non existing volume
+
+        volname = "non_existent_subvolume"
+
+        # try, create subvolume
+        try:
+            self._fs_cmd("subvolume", "create", volname, "subvolume_1")
```

Comment:
I think you can just loop over the different operations rather than unroll it:

````
for op in ("create", "rm", ...):
  try: ...
````

Reply:
Done